engine/src/main/java/org/hibernate/validator/internal/constraintvalidators/hv/EmailValidator.java

```diff
@@ -31,9 +31,11 @@
  * @author Hardy Ferentschik
  */
 public class EmailValidator implements ConstraintValidator<Email, CharSequence> {
-	private static String ATOM = "[a-z0-9!#$%&'*+/=?^_`{|}~-]";
-	private static String DOMAIN = ATOM + "+(\\." + ATOM + "+)*";
-	private static String IP_DOMAIN = "\\[[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\]";
+	private static final String ATOM = "[a-z0-9!#$%&'*+/=?^_`{|}~-]";
+	private static final String DOMAIN = ATOM + "+(\\." + ATOM + "+)*";
+	private static final String IP_DOMAIN = "\\[[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\]";
```

Comment:
So there are no changes to the actual expressions, right? Only the "final" has been added?